### ISSUE TITLE
TopologyTestApp requires a GUI to run

### ISSUE THREAD

**Issue by strk** [locationtech/jts#109]
On a unix system, running the following:
```
java org.locationtech.jtstest.testrunner.TopologyTestApp -files test.xml
```

Requires a working X connection:
```
No X11 DISPLAY variable was set, but this program performed an operation which requires it.
        at java.awt.GraphicsEnvironment.checkHeadless(GraphicsEnvironment.java:204)
        at java.awt.Window.<init>(Window.java:536)
        at java.awt.Frame.<init>(Frame.java:420)
        at java.awt.Frame.<init>(Frame.java:385)
        at javax.swing.JFrame.<init>(JFrame.java:189)
        at org.locationtech.jtstest.testrunner.TopologyTestApp.<init>(TopologyTestApp.java:188)
        at org.locationtech.jtstest.testrunner.TopologyTestApp.<init>(TopologyTestApp.java:174)
        at org.locationtech.jtstest.testrunner.TopologyTestApp.main(TopologyTestApp.java:213)
```

**Comment by dr-jts** [comment]
That's because TopologyTestApp extends JFrame.  This is a legacy from when the TopologyTestApp was run as a GUI app.  That functionality is long dead, so this could be fixed so that the entry point is a pure command-line utility.

**Comment by dr-jts** [comment]
Try this out:  https://github.com/dr-jts/jts/tree/feature-testrunner

It contains a JTSTestRunnerCmd class, with no Swing dependency.

**Comment by dr-jts** [comment]
I have updated the Maven build to create a runnable jar for the TestRunner, and also the testrunner.bat script to run it.  testrunner.sh left as an exercise for the student for now - updates welcome.

To come: default the test files to be run to the standard JTS test XML files

**Comment by dr-jts** [comment]
Fixed by #114
